src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/Offender.java

```diff
@@ -195,8 +188,10 @@
     @JoinColumn(name = "PARTITION_AREA_ID")
     private PartitionArea partitionArea;
 
-    @JoinColumn(name = "OFFENDER_ID")
-    @OneToMany
+    @OneToMany(mappedBy = "offenderId", fetch = FetchType.LAZY)
```

Comment:
LAZY is default fetch

Reply:
You are right - I've altered this one to remove the specific fetch type. For info, for @ManyToOne relationships it default to FetchType.EAGER.